api/src/main/java/org/openmrs/ui/framework/UiUtils.java

```diff
@@ -586,4 +586,30 @@ public Locale getLocale() {
 	public void setLocale(Locale locale) {
 		this.locale = locale;
 	}
+
+	/**
+	 * @return the value of the Global Propriety Handle Timezones 
+	 */
```

Comment:
I think this JavaDoc can go. Just to be consistent with the rest of the file and because this method should be clear enough at first sight.